src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -68,7 +68,10 @@ class CGroup
     static void Initialize()
     {
         s_cgroup_version = FindCGroupVersion();
-        FindCGroupPath(s_cgroup_version == 1 ? &IsCGroup1MemorySubsystem : nullptr, &s_memory_cgroup_path, &s_memory_cgroup_hierarchy_mount);
+        if (s_cgroup_version != 0)
```

Comment:
Can we document what this means. Do we not support version `0` or does `0` mean invalid/not supported?

Reply:
it's documented on line 62

Reply:
Perfect. Thanks.